clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1002,6 +1019,13 @@ raw2trace_t::process_header(raw2trace_thread_data_t *tdata)
     create_essential_header_entries(buf, version, tdata->file_type, tid, pid);
     buf += trace_metadata_writer_t::write_marker(buf, TRACE_MARKER_TYPE_CACHE_LINE_SIZE,
                                                  header.cache_line_size);
+    if (header.vector_length_bytes > 0) {
+#ifdef AARCH64
+        dr_set_sve_vector_length(header.vector_length_bytes * 8);
```

Comment:
Please update the opcode_mix and invariant_checker tools to set this as well so their decoding will be accurate.

Reply:
Done.